main.cpp

```diff
@@ -2353,6 +2355,12 @@ int main(int argc, char **argv) {
                     if (selected_cmd->force_requires_pre_reboot()) {
                         // we reboot into BOOTSEL mode and disable MSC interface (the 1 here)
```

Comment:
Probably makes sense to move this comment to immediately above the `reboot_device(to_reboot, true, 1);` line?